dev/com.ibm.ws.microprofile.reactive.streams.operators_fat/publish/servers/ReactiveStreamsTestServer/server.xml

```diff
@@ -13,7 +13,6 @@
 	</featureManager>
 
    <application id="mpReactiveStreamsFAT" name="ReactiveStreamsTest" type="war" location="ReactiveStreamsTest.war">
```

Comment:
now that you don't need the classloader bit, do you need the application entry at all? can it go back to using dropins? Fine if that's a separate PR